src/sqlancer/sqlite3/SQLite3Provider.java

```diff
@@ -49,12 +55,26 @@ public class SQLite3Provider extends SQLProviderAdapter<SQLite3GlobalState, SQLi
     public static boolean allowFloatingPointFp = true;
     public static boolean mustKnowResult;
 
+    private Map<String, String> queryPlanPool;
+
+    // Data structures for QPG
+    private static double[] weightedAverageReward = new double[Action.values().length];
+    private int currentSelectRewards;
+    private int currentSelectCounts;
+    private int currentMutationOperator;
```

Comment:
I think we could perform the below initializations already here? Like `currentMutationOperator = -1`.